### PR TITLE
Event drilldown main page + Organiser Navbar

### PR DESCRIPTION
**Pull request by brian2w**
## Why was your change needed?

We need to stub out the frontend of organiser platform to be able to start connecting it to the backend database.

## What was Changed?

Made both Organiser Navbar and Event Drilldown Pages

## Any extra notes for reviewers to know?

Write any extra notes you may have for the PR.
This may include specific code paths that need extra scrutiny, or how to review a specific portion of code.

### CODE REVIEW

File: frontend/app/organiser/event/[id]/page.tsx [hunk 1]
```diff
@@ -1,8 +1,58 @@
...
+export default function EventPage({ params }: EventPageProps) {
+  const [currSidebarPage, setCurrSidebarPage] = useState("Details");
+  const [eventData, setEventData] = useState<EventData>();
+
+  const eventId: EventId = params.id;
+  useEffect(() => {
+    getEventById(eventId)
+      .then((event) => setEventData(event))
+      .catch((error) => {
+        console.log("error:", error);
```
Comment (41dan) [inline frontend/app/organiser/event/[id]/page.tsx:33]: remove

**Review by 41dan — COMMENTED** [review]
image
change middle edit icon to match size

**Review by 41dan — APPROVED** [review]

File: frontend/app/organiser/event/[id]/page.tsx (hunk 1 above)
Comment (brian2w) [reply frontend/app/organiser/event/[id]/page.tsx]: I've addressed this by commenting out this piece of code for the moment.

**Review by 41dan — APPROVED** [review]
lgtm

**Pull request merged**